src/Components/ModalDialog.razor

```diff
@@ -0,0 +1,58 @@
+<div class="modal fade show" id="confirmationModal" style="display:block; background-color: rgba(10,10,10,.8);" aria-modal="true" role="dialog">
```

Comment:
This is nice. A reusable modal dialog component. Great idea! :).

Reply:
fixed in 23ec94e. Due to this i also ~~accidentally~~ intentionally fixed the sorting buttons of the quickgrid :)